cypress/e2e/01-personalia_spec.cy.ts

```diff
@@ -0,0 +1,34 @@
+describe("Soknad step 1: Personal Information", () => {
+    let isFirstTest = true;
+
+    beforeEach(() => {
+        if (isFirstTest) {
+            cy.startNewSoknad();
+            cy.get('[data-testid="personalia-infotekst"]').should("be.visible");
+        } else {
+            cy.continueSoknad();
+        }
+        isFirstTest = false;
+    });
+
+    it("Should be able to edit the phone number field", () => {
+        cy.get('[data-testid="telefon-endreknapp"]').click();
+        cy.get('input[name="brukerutfyltVerdi"]').type("40506070");
+        cy.get('[data-testid="lagre-telefonnummer"]').click();
+    });
+
+    it("Should be able to edit the kontonummer field", () => {
+        cy.get('[data-testid="kontonummer-endreknapp"]').click();
+        cy.get('input[name="brukerutfyltVerdi"]').type("0000 11 22223");
+        cy.get('[data-testid="lagre-kontonummer"]').click();
+    });
+
+    it("Should be able to pick address", () => {
+        cy.get('[data-testid="addresse-valg"]').click();
+    });
+
+    it("Should be able to continue to the next page", () => {
+        cy.contains("Neste steg").click();
+        cy.get('[data-testid="skjemasteg-2-heading"]').should("be.visible");
+    });
+});
```

Comment:
Denne testen kjører, men man burde nok gjøre noe for å være mer sikker på at den faktisk fortsetter på søknaden den oppretter i første steg

Reply:
Gjerne også mocke APIet og bekrefte at den viser riktig data og setter verdiene riktig